channeldb/db.go

```diff
@@ -520,9 +526,6 @@ type ChannelStateDB struct {
 	// linkNodeDB separates all DB operations on LinkNodes.
 	linkNodeDB *LinkNodeDB
 
-	// nodeAnnouncementDB seperates all DB operations on NodeAnnouncements.
-	nodeAnnouncementDb *NodeAnnouncementDB
-
```

Comment:
this commit is removing a bunch of the code added in the previous commit - so it makes it very hard to grok what is actually happening in the PR. Maybe take some time to structure the commits in a more story-telling fashion 🙏

Reply:
Thanks for taking the time to review. I will restructure the commits to facilitate the review process.